src/aiida/storage/psql_dos/orm/querybuilder/main.py

```diff
@@ -593,7 +592,7 @@ def get_filter_expr(
                 )
             else:
                 if column is None:
-                    if (alias is None) and (column_name is None):
+                    if alias is None or column_name is None:
```

Comment:
This was a logic bug, per the `get_column` type signature, neither alias **nor** column name can be None.